Clicker/ViewControllers/LoginViewController.swift

```diff
@@ -35,16 +65,33 @@ class LoginViewController: UIViewController, GIDSignInUIDelegate {
     
     // MARK: - CONSTRAINTS
     func setupConstraints() {
-        appLogoImage.snp.makeConstraints { make in
-            make.center.equalToSuperview()
-            make.width.height.equalTo(70)
+        
+        welcomeLabel.snp.makeConstraints { make in
+            make.width.equalTo(249.5)
+            make.height.equalTo(31.5)
+            make.top.equalToSuperview().offset(194.5)
+            make.centerX.equalToSuperview()
+        }
+        
+        appNameLabel.snp.makeConstraints { make in
+            make.width.equalTo(welcomeLabel.snp.width)
+            make.height.equalTo(71.5)
+            make.top.equalTo(welcomeLabel.snp.bottom)
+            make.centerX.equalToSuperview()
+        }
+        
+        pronunciationLabel.snp.makeConstraints { make in
+            make.width.equalTo(welcomeLabel.snp.width)
+            make.height.equalTo(19)
+            make.top.equalTo(appNameLabel.snp.bottom).offset(14.5)
+            make.centerX.equalToSuperview()
         }
         
         signInButton.snp.makeConstraints { make in
-            make.bottom.equalToSuperview().offset(-60)
+            make.width.equalTo(200)
+            make.height.equalTo(40)
+            make.top.equalTo(pronunciationLabel.snp.bottom).offset(28)
```

Comment:
same with all these constants

Reply:
Ok, thanks for pointing this out! I'll make those fixes.